go/noise/gaussian_noise_test.go

```diff
@@ -856,6 +896,30 @@ func TestComputeConfidenceIntervalFloat64Gaussian(t *testing.T) {
 			want:            ConfidenceInterval{-96.6140883158, 236.6140883158},
 			wantErr:         false,
 		},
+		// float64 precision should be accurate for abs(noisedX) < 2^54
```

Comment:
I am not sure I understand the purpose of this comment.

Reply:
This was mainly to test that high noised values can be presented accurately in the float64 version of the function as long as the noisedX < 2^54

Reply:
I see. Makes sense. Maybe something like "testing that bounds are accurate for abs(bound) < 2^53" is more descriptive.